governance/votes/Executive vote - May 06, 2024.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Template - [Executive Vote] Approve Coinbase Custody (RWA014-A) DAO Resolution - May 06, 2024
+summary: Approve Coinbase Custody (RWA014-A) DAO Resolution that extends the Coinbase Custody Rewards Program for six months.
+date: 2024-05-06T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Approve Coinbase Custody (RWA014-A) DAO Resolution - May 06, 2024
+
+The Governance Facilitators, dewix and Sidestream have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- A DAO Resolution pertaining to RWA014-A (Coinbase Custody) will be approved, as detailed below.
+
+**Voting for this executive proposal will place your MKR in support of the actions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
```

Comment:
We should update the template

```suggestion
Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **30 hours**.
```